code/datums/status_effects/neutral.dm

```diff
@@ -342,3 +342,188 @@
 	tick_interval = INFINITY
 	status_type = STATUS_EFFECT_REFRESH
 	alert_type = null
+
+/datum/status_effect/eigenstasium
+	id = "eigenstasium"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = null
+	///So we know what cycle we're in during the status
+	var/current_cycle = -250 //Consider it your stability
+	///The addiction looper for addiction stage 3
+	var/phase_3_cycle = -0 //start off delayed
+	///Your clone from another reality
+	var/mob/living/carbon/alt_clone = null
+	///If we display the stabilised message or not
+	var/stable_message = FALSE
+
+///Convert this effect to process slowly instead
+/datum/status_effect/eigenstasium/on_creation(mob/living/new_owner, ...)
+	. = ..()
+	STOP_PROCESSING(SSfastprocess, src)
+	START_PROCESSING(SSprocessing, src) //this lasts a while, so SSfastprocess isn't needed.
+
+/datum/status_effect/eigenstasium/Destroy()
+	QDEL_NULL(alt_clone)
+	. = ..()
+	STOP_PROCESSING(SSprocessing, src)
+
+
+/datum/status_effect/eigenstasium/tick()
+	. = ..()
+	//This stuff runs every cycle
+	if(prob(5))
+		do_sparks(5, FALSE, owner)
+
+	//If we have a reagent that blocks the effects
+	var/block_effects = FALSE
+	if(owner.has_reagent(/datum/reagent/bluespace))
+		current_cycle = max(-250, (current_cycle - 7.5)) //cap to -250
+		block_effects = TRUE
+	if(owner.has_reagent(/datum/reagent/stabilizing_agent))
+		current_cycle = max(-250, (current_cycle - 5))
+		block_effects = TRUE
+	var/datum/reagent/eigen = owner.has_reagent(/datum/reagent/eigenstate)
```

Comment:
Why was this done as a status effect?

Reply:
Because the old addiction mechanics were removed and I talked to the creator of the new rework and they said they wanted to avoid single use addictions.

Reply:
There's a few benefits of doing it this way too - it ticks faster, is more self contained and lets you reverse the effects by taking one of the 3 reagents.